src/classes/ADDR_Addresses_UTIL.cls

```diff
@@ -46,255 +46,298 @@ public with sharing class ADDR_Addresses_UTIL {
     public static Boolean isAddressChanged(Address__c addrNew, Address__c addrOld, Boolean includeAddressType) {
-        if (addrNew == null || addrOld == null) 
-           return false;        
-        return (
-            addrNew.Default_Address__c != addrOld.Default_Address__c ||
-           !equalsCaseSensitive(addrNew.MailingStreet__c, addrOld.MailingStreet__c) ||
-           !equalsCaseSensitive(addrNew.MailingStreet2__c, addrOld.MailingStreet2__c) ||
-           !equalsCaseSensitive(addrNew.MailingCity__c, addrOld.MailingCity__c) ||
-           !equalsCaseSensitive(addrNew.MailingState__c, addrOld.MailingState__c) ||
-           !equalsCaseSensitive(addrNew.MailingPostalCode__c, addrOld.MailingPostalCode__c) ||
-           !equalsCaseSensitive(addrNew.MailingCountry__c, addrOld.MailingCountry__c) ||
-           (includeAddressType && (!equalsCaseSensitive(addrNew.Address_Type__c, addrOld.Address_Type__c))));
+        if (addrNew == Null || addrOld == Null) {
+            return False; 
+        }
+                    
+        return (addrNew.Default_Address__c != addrOld.Default_Address__c ||
+                !equalsCaseSensitive(addrNew.MailingStreet__c, addrOld.MailingStreet__c) ||
+                !equalsCaseSensitive(addrNew.MailingStreet2__c, addrOld.MailingStreet2__c) ||
+                !equalsCaseSensitive(addrNew.MailingCity__c, addrOld.MailingCity__c) ||
+                !equalsCaseSensitive(addrNew.MailingCounty__c, addrOld.MailingCounty__c) ||
+                !equalsCaseSensitive(addrNew.MailingState__c, addrOld.MailingState__c) ||
+                !equalsCaseSensitive(addrNew.MailingPostalCode__c, addrOld.MailingPostalCode__c) ||
+                !equalsCaseSensitive(addrNew.MailingCountry__c, addrOld.MailingCountry__c) ||
+                (includeAddressType && (!equalsCaseSensitive(addrNew.Address_Type__c, addrOld.Address_Type__c)))
+                );
     }
     
     /*******************************************************************************************************
-    * @description compares two contacts' addresses
-    * @param con1 a Contact
-    * @param con2 a Contact
-    * @return Boolean. true if the Mailing Address fields have changed 
+    * @description Compares two Contact records' address fields. 
+    * @param con1 a Contact record. 
+    * @param con2 a Contact record. 
+    * @return Boolean. True if the Mailing Address fields have changed. 
     ********************************************************************************************************/
     public static Boolean isContactAddressChanged(Contact con1, Contact con2) {
-        // if both null, no change
-        if (con1 == null && con2 == null) 
-           return false;
+        if (con1 == Null && con2 == Null) {
+            return false;
+        }
 
-        // if one null, make sure other has at least one address field set
```

Comment:
Looks like we are getting rid of this comment. I found the return statement is a bit confusing to understand. Maybe instead of removing it, we can make the comment a bit clearer?